ehitchhiking-ui/src/app/shared/components/active-trip/active-trip.component.ts

```diff
@@ -1,25 +1,27 @@
-import {Component, Input, OnInit} from '@angular/core';
-import {ActiveTrip} from './active-trip';
-import {UserState} from '@shared/enums/UserState';
-import {DEFAULT_MAT_DIALOG_CLASS, MAT_DIALOG_WIDTH_SM} from '@shared/constants/modal-constants';
-import {MatDialog} from '@angular/material';
-import {ConfirmationModalComponent} from '@shared/modals/confirmation-modal/confirmation-modal.component';
-import {ActiveTripsModalService} from '@shared/components/active-trips-modal/active-trips-modal.service';
+import { Component, EventEmitter, Input, OnInit, Output } from "@angular/core";
+import { UserState } from "@shared/enums/UserState";
+import { DEFAULT_MAT_DIALOG_CLASS, MAT_DIALOG_WIDTH_SM } from "@shared/constants/modal-constants";
+import { MatDialog } from "@angular/material";
+import { ConfirmationModalComponent } from "@shared/modals/confirmation-modal/confirmation-modal.component";
+import { ActiveTrip } from "@shared/models/active-trip";
+import { ActiveTripsApiService } from "@shared/services/api.services/active-trips.api.service";
 
 @Component({
 	selector: 'app-active-trip',
 	templateUrl: './active-trip.component.html',
 	styleUrls: ['./active-trip.component.sass'],
 })
 export class ActiveTripComponent implements OnInit {
-	constructor(public dialog: MatDialog, public tripService: ActiveTripsModalService) {}
+	constructor(public dialog: MatDialog, public apiService: ActiveTripsApiService) {}
```

Comment:
pls rename to activeTripsApiService. You can import here thousands of  different services and api service here, hence we should be sure that variable name should be unique. 

activeTripsApiService should be private property

Reply:
fixed